bin/generate

```diff
@@ -71,11 +71,11 @@ let curries = (start...limit+1).map { curryDefinitionGenerator(arguments: $0) }
 let output = curries.joined(separator: "\n\n") + "\n"
 
 let outputPath = "Source/Curry.swift"
-let currentPath = URL(fileURLWithPath: FileManager.default.currentDirectoryPath)
-let currySwiftPath = currentPath.appendingPathComponent(outputPath)
+let parentPath = URL(fileURLWithPath: FileManager.default.currentDirectoryPath).deletingLastPathComponent()
```

Comment:
I'm afraid this approach could backfire. In fact, if I try it on my machine, where the repo is located at

```
~/Developer/open-source/Curry
```

what I get is (_indented for convenience_)

```
~/D/o/Curry (remotes/zhongwuzw/master) ❯❯❯ rm Source/Curry.swift
remove Source/Curry.swift? y

~/D/o/Curry (remotes/zhongwuzw/master ●1) ❯❯❯ git status
HEAD detached at 48fca58
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    Source/Curry.swift

no changes added to commit (use "git add" and/or "git commit -a")

~/D/o/Curry (remotes/zhongwuzw/master) ❯❯❯ bin/generate
Generating 💬
An error occurred while saving the generated functions. Error: 
	Error Domain=NSCocoaErrorDomain
	Code=4 "The folder “Curry.swift” doesn’t exist."
	UserInfo= { 
		NSFilePath=/Users/gio/Developer/open-source/Source/Curry.swift,
		NSUserStringVariant=Folder,
		NSUnderlyingError=0x7faac064ceb0 {
			Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"
		}
	}
```

On the other hand if I try the same combo on `master (386b060)` it works ok.

```
~/D/o/Curry (master) ❯❯❯ rm Source/Curry.swift
remove Source/Curry.swift? y

~/D/o/Curry (master ●1) ❯❯❯ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    Source/Curry.swift

no changes added to commit (use "git add" and/or "git commit -a")

~/D/o/Curry (master ●1) ❯❯❯ bin/generate
Generating 💬
Done, curry functions files written at Source/Curry.swift 👍

~/D/o/Curry (master) ❯❯❯ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```